02_housing/1a_housing_place.R

```diff
@@ -294,7 +293,7 @@ households_2022 <- left_join(microdata_housing, place_income_limits_2022, by=c("
 # for renter and owner subgroups, this is just the population of renters or owners at each income level
 
 # create new variable 'Affordable80AMI' and 'Below80AMI' for HH below 80% of area median income (L80_4 and OWNERSHP)
-# if OWNERSHP is not equal to 1 or 2, leave as NA
+# if OWNERSHP is not equal to 1 or 2, leave as NA 
```

Comment:
We have an issue on the case_when code:

```
Affordable80AMI_all =
           case_when(# deal with cases when RENTGRS and OWNCOST are 0
             RENTGRS == 0 ~ 0,
             OWNERSHP==2 & ((RENTGRS*12)<=(l80_4*0.30)) ~ 1,
             OWNERSHP==2 & ((RENTGRS*12)>(l80_4*0.30)) ~ 0,
             OWNERSHP==1 & ((OWNCOST*12)<=(l80_4*0.30)) ~ 1,
             OWNERSHP==1 & ((OWNCOST*12)>(l80_4*0.30)) ~ 0)
```

I believe that if the record is for an owner (OWNERSHP==1) that it is a zero on the variable RENTGRS. So essentially you are missing all ownership cases because you are making them zero off the bat. This also applies to the 50 and 30 AMI code sections.

It looks like the county code is correct. However I feel like a better and safer way to code this would be to calculate Affordable80AMI_renter and Affordable80AMI_owner first, then calculate Affordable80AMI_all with the formula: if Affordable80AMI_renter==1 OR Affordable80AMI_owner==1. 

This needs to be fixed here as it is used in the 2a and 2b scripts.

Reply:
However you decide to fix it, I feel like we should make sure it is coded the same way on the county script for consistency.

Reply:
IPUMS loads data labels, which can be converted and used directly. FWIW, I think this limits issues with mixing up 0/1. It may not have exactly solved this problem, but it may have been clearer.

Reply:
OK I re-checked all of the variables to make sure the _renter and _owner version are the same as the _all and it check out. Going to resolve this now.